### PR TITLE
Vibrokatana

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

image

Adds an upgrade to katana for guild so that people actually have a reason to ever make it.
<hr>
</details>

## Changelog
:cl:
add: Vibrokatana
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
why though, soteria already has muramasa

**Comment by cdb-is-not-good** [comment]
Why...? 
Also these sprites are....not at all up to our standard.

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 1]
```diff
@@ -267,6 +267,19 @@
 	else
 		overlays += "[icon_state]_power_off"
 
+/obj/item/tool/sword/katana/vibro
+	name = "artificer vibrokatana"
+	desc = "A modern katana made by a madman painted all over with red and gloss. Just being near it makes your ears ring from it's sharp edge cutting up against the air."
```
Comment (elgeonmb) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:272]: should have a comma between "madman" and "painted". wrong form of "it's"-- should be "its".

**Comment by lolman360** [comment]
muramasa isnt foldable so yes this is basically muramasa 2: guild edition

**Pull request closed without merging**